org-code-codebuilderapp/app/build.gradle

```diff
@@ -0,0 +1,45 @@
+/*
+ * Usage:
+ * gradle build -> builds the project
+ * gradle bootRun -> sets up a local server -> can be accessed at localhost:8080
+ *
+ * To get your changes into the docker container, do the following
+ * gradle build
+ * cp app/build/libs/app-0.0.1-SNAPSHOT.jar dockerContainer
+ * cd dockerContainer
+ * docker build .
+ *
+ * To publish that docker container, do the following
+ * docker tag <id> jmkulwik/test-fargate-websocket:<tag>
+ * docker push jmkulwik/test-fargate-websocket:<tag>
+ */
+
+plugins {
+  id 'org.springframework.boot' version '2.4.2'
+  id 'io.spring.dependency-management' version '1.0.11.RELEASE'
+  id 'java'
+}
+
+group = 'org.code'
+version = '0.0.1-SNAPSHOT'
```

Comment:
My memory is a little fuzzy, but I think there are some implications of using a SNAPSHOT version and also having the version string not change from build to build.  Maybe just double-check that the resulting behavior is what you want?

Reply:
Ooohhh thanks. I'll look into that.

Reply:
✔️